ocaml/xapi-cli-server/cli_frontend.ml

```diff
@@ -3639,8 +3639,15 @@ let rec cmdtable_data : (string * cmd_spec) list =
     )
   ; ( "repository-introduce"
     , {
-        reqd= ["name-label"; "binary-url"; "source-url"; "update"]
-      ; optn= ["name-description"; "gpgkey-path"]
+        reqd= ["name-label"; "update"]
```

Comment:
I think as we can not introduce a repository with only "name-label" and "update", we can not have only the 2 as required parameter. At least `origin` is also required?